src/Apps/ArtQuiz/__tests__/ArtQuizResultsLoader.jest.tsx

```diff
@@ -1,22 +1,21 @@
 import React from "react"
-import { render, screen } from "@testing-library/react"
+import { screen } from "@testing-library/react"
 import {
   ArtQuizResultsLoader,
   waitTime,
 } from "Apps/ArtQuiz/Components/ArtQuizResultsLoader"
 import { act } from "react-dom/test-utils"
+import { render } from "DevTools/setupTestWrapper"
```

Comment:
I think you're right that we need to globally mock this. This is painful to have to know and is hidden. This is fine for now, but I'll make a ticket to actually fix this.

Reply:
Resolved here: https://github.com/artsy/force/pull/11238 — will have to rebase.